{"name": "Принять заказ. Попытка принять заказ с некорректным id курьера возвращает корректный статус-код", "status": "passed", "description": "тест ручки \"Принять заказ\"", "start": 1724853724671, "stop": 1724853724863, "uuid": "24fa5b26-61b4-43a7-b6de-d8edf45f8214", "historyId": "be8453279860753e7198b6bf0926f9ec", "testCaseId": "be8453279860753e7198b6bf0926f9ec", "fullName": "tests.TestAcceptOrder#test_try_accept_order_with_non_existing_courier_id_returns_correct_status_code", "labels": [{"name": "suite", "value": "tests"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "FanatovPC"}, {"name": "thread", "value": "9332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests"}]}